home.admin/config.scripts/bitcoin.install.sh

```diff
@@ -241,12 +241,13 @@ signet.addnode=nsgyo7begau4yecc46ljfecaykyzszcseapxmtu6adrfagfrrzrlngyd.onion:38
 
   removeParallelService
 
-  if [ ${CHAIN} = mainnet ];then
-    sudo cp /home/admin/assets/bitcoind.service /etc/systemd/system/bitcoind.service
-  else
-    # /etc/systemd/system/${prefix}bitcoind.service
-    # based on https://github.com/bitcoin/bitcoin/blob/master/contrib/init/bitcoind.service
-    echo "
+  # /etc/systemd/system/${prefix}bitcoind.service
+  # based on https://github.com/bitcoin/bitcoin/blob/master/contrib/init/bitcoind.service
+  chainparamter=""
+  if [ "${CHAIN}" != "mainnet" ]; then
+    chainparamter="-${CHAIN}"
```

Comment:
should this be:
```suggestion
  chainparameter=""
  if [ "${CHAIN}" != "mainnet" ]; then
    chainparameter="-${CHAIN}"
```
?